helpdesk_mgmt/views/helpdesk_ticket_team_view.xml

```diff
@@ -31,11 +31,54 @@
                         <label for="name" class="oe_edit_only"/>
                         <h1><field name="name" required="1"/></h1>
                     </div>
-                    <group name="main">
-                        <field name="user_ids" widget="many2many_tags"/>
-                        <field name="company_id" groups="base.group_multi_company"/>
+                    <group>
+                        <group name="left">
+                            <field name="active" invisible="1"/>
+                            <field name="user_id" domain="[('share', '=', False)]"/>
+                            <label for="alias_name" string="Email Alias"/>
+                            <div class="oe_inline" name="alias_def">
+                                <field name="alias_id" class="oe_read_only oe_inline"
+                                    string="Email Alias" required="0"/>
+                                <div class="oe_edit_only oe_inline" name="edit_alias" style="display: inline;" >
+                                    <field name="alias_name" class="oe_inline"/>@<field name="alias_domain" class="oe_inline" readonly="1"/>
+                                    <button  icon="fa-arrow-right" type="action" name="%(base_setup.action_general_configuration)d" string="Configure domain name" class="btn-link" attrs="{'invisible': [('alias_domain', '!=', False)]}"/>
+                                </div>
+                            </div>
+                            <field name="alias_contact" class="oe_inline"
+                                string="Accept Emails From" />
+                            <field name="alias_user_id" string="Assign to"/>
+                            <field name="company_id" options="{'no_create': True}" groups="base.group_multi_company"/>
+                        </group>
+                        <group name="right">
+                        </group>
                     </group>
+                    <notebook>
+                        <page name="members" string="Team Members" >
```

Comment:
And how is this related to alias?

Reply:
Idem that user_ids... It is not related with alias... other PR, no? :)

Reply:
Yes!

Reply:
Done